src/utils/formatTime.js

```diff
@@ -40,4 +40,13 @@ function formatCountdown (seconds) {
   }
 }
 
-export { formatDate, formatTime, formatToUTC, formatCountdown }
+function calculateTime (session) {
+  const currentTime = new Date().getTime()
+  const sessionStartTime = new Date(session.start).getTime()
+  const sessionEndTime = new Date(session.end).getTime()
+  let countdown
```

Comment:
I would rename this to sessionTime, since the countdown is really the thing you are returning, not this variable. This variable is either the start or end session time used to calculate the countdown.